### ISSUE TITLE
Vendor security review of Screendoor

### ISSUE THREAD

**Issue by simonwex** [MozillaFoundation/mofo-devops#321]
[Screendoor link](https://www.dobt.co/screendoor/)

**Comment by cadecairos** [comment]
@Melechuga 

QQ: are you going to be the main user of screendoor? This is a guess based on your CC in the email thread... I'm not sure if Jonathan spoke with you, but we were hoping you could join a [Rapid Risk Assessment](https://wiki.mozilla.org/Security/Risk_management/Rapid_Risk_Assessment) for screendoor next week (Jonathan will schedule)

also see: https://bugzilla.mozilla.org/show_bug.cgi?id=1266204

**Comment by Melechuga** [comment]
Hi @cadecairos. Yes, I will be the main user of Screendoor; we have a number of application/RFPs throughout the year, and would like to use their services.  I'd be happy to join the RRA (my first!) and will read up on all the docs provided on the wiki. What should I prepare for the call?

**Comment by cadecairos** [comment]
@Melechuga If you can bring an understanding of how we're going to use screendoor as well as whose data, and what data we'll be storing within it, we should be all set.

I've collected as much information about Screendoor's systems and security as they've made available to me through email, and will have it available for Jonathan (He works on MoCo Enterprise Information Security team, and will be chairing the RRA)

**Comment by cadecairos** [comment]
TL;DR: Go ahead and use Screendoor, @Melechuga to follow up on the risk report bug that Jonathan will file. 

We've [completed an RRA of screendoor](https://bugzilla.mozilla.org/show_bug.cgi?id=1266204), and based on that, we feel there's not enough risk at this time to warrant further investigation into the system by The MoCo security team. Should it become a pain in our back down the road, we will re-evaluate this position.
